force-app/main/default/lwc/participantSelector/participantSelector.js

```diff
@@ -194,9 +211,25 @@ export default class ParticipantSelector extends LightningElement {
         return this.offset < this.filteredEngagements.length;
     }
 
+    handleNewParticipantClick() {
+        const newParticipant = this.template.querySelector("c-new-program-engagement");
+        newParticipant.showModal();
+    }
+
+    handleNewParticipantSuccess(event) {
+        this.catchNewPE(event.detail);
+    }
+
+    async catchNewPE(id) {
```

Comment:
What do you think about processNewParticipant or something like that? catchNewPE feels like a deviation from the rest of the naming.